src/main/java/com/gw/exercises/task1/helloName.java

```diff
@@ -0,0 +1,33 @@
+package com.gw.exercises.task1;
+
+public class helloName {
+
+    // All className we need to start Capitalised
+    //it suitable to use "_" and numbers
+    // i can to start naming variable from $ and _
+    // camelCaseUsedToSeparateWords
```

Comment:
What the reason for these comments?

Reply:
Just a note for myself. Will discard it.